cc_licenses/templates/includes/use_of_licenses.html

```diff
@@ -43,12 +43,15 @@ <h4 class="level is-vcentered b-header">{% trans "Considerations for the public"
 </div>
 
 <style>
-  .has-background-info-light .divider {
+  [dir="rtl"] .has-background-info-light .divider {
+      margin-right: -2rem;
+  }
+  [dir="ltr"] .has-background-info-light .divider {
       margin-left: -2rem;
+  }
+
```

Comment:
@zackkrida , we added this css because the divider in `use_of_licenses.html` section is misaligned when reading languages that are right to left. Just want to make you aware of it.